src/api/spec/models/user_spec.rb

```diff
@@ -623,4 +623,48 @@
       end
     end
   end
+
+  describe '#can_modify?' do
```

Comment:
There are also some leftovers in `test/unit/user_test.rb` you should retire IMHO